### PR TITLE
Moon: increase gravity a little

### PR DESCRIPTION
**Pull request by JostP**
Changes gravity due to reports of players that 0.16 gravity is a bit annoying.

### CODE REVIEW

**Review by Panquesito7 — COMMENTED** [review]
I've tested it; to be honest, it isn't as fun as it was before. Maybe decrease a little bit the gravity?

File: moon/map.conf [hunk 1]
```diff
@@ -12,9 +12,9 @@ team.2 = blue
...
 
-phys_gravity = 0.16
-phys_speed = 1.2
-phys_jump = 0.8
+phys_gravity = 0.3
+phys_speed = 1.25
+phys_jump = 1
 
 base_node = ctf_map:gravel
 initial_stuff = default:shovel_steel,shooter:pistol
```
Comment (moreavy) [inline moon/map.conf:20]:
```suggestion
initial_stuff = default:shovel_steel,shooter:pistol,default:torch 3
```

**Review by moreavy — COMMENTED** [review]
Maybe adding torches in `initial_stuff` would be a good idea though.

**Comment by moreavy** [comment]
I suggest the `decision needed ⚖️` label can be used here as @Panquesito7 and @moreavy have tested and it works fine. But what we need is just a final decision right now or maybe a change the the moon's `map.conf` file.

**Comment by JostP** [comment]
> I've tested it; to be honest, it isn't as fun as it was before. Maybe decrease a little bit the gravity?

Thanks for testing. Yes, we haven't decided if this will be merged, that's why we need many people to tell their opinion.

**Review by olliy1or — APPROVED** [review]

**Review by moreavy — CHANGES_REQUESTED** [review]
Not as fun as the old moon. Less Gravity makes the moon realistic. I didn't like the increased gravity of the moon.

**Review by MoreKat — COMMENTED** [review]
Tested it and I wouldn't change it. The reduced gravity is what makes the map unique, and I don't think this would have a significant impact on people's opinion about the map.

**Review by Toby1710 — APPROVED** [review]

**Comment by Panquesito7** [comment]
> Maybe adding torches in `initial_stuff` would be a good idea though.

This should be made in another PR.

**Comment by JostP** [comment]
Will deal with this later.
I have some plans to improve moon map one day, currently it has some issues, on of them long matches.

**Pull request closed without merging**